src/components/search-input/search-input.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import "./search-input.scss";
+import { FaSearch } from "react-icons/fa";
+
+export const SearchInput = ({
+  hasIcon = true,
+  onValueChange,
+  placeholder = "cauta",
+  value
+}) => {
+  const [currentValue, setCurrentValue] = useState(0);
+  return (
+    <div className={`__search-input-container ${hasIcon ? "__has_icon" : ""}`}>
+      <input
+        placeholder={placeholder}
+        value={value || currentValue || ""}
+        onChange={event => {
+          if (onValueChange) {
+            onValueChange(event);
```

Comment:
If we want to pass the event in the callback, we should call this `onChange`.

I think for the scope of this component, we can have:
```suggestion
            onValueChange(event.target.value);
```